lib/lib-utils/pom.xml

```diff
@@ -47,7 +47,10 @@
 			<artifactId>mockito-junit-jupiter</artifactId>
 			<scope>test</scope>
 		</dependency>
-
+		<dependency>
```

Comment:
I do not think we need to use gson, the JSON-java should already be sufficient to read from the stream.